images/base/Dockerfile

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# runc (v1.0.0-rc93-pre)
+FROM golang:1.15-alpine AS runc
+RUN apk add --no-cache git build-base libseccomp-dev
+RUN git clone -q https://github.com/opencontainers/runc.git /go/src/github.com/opencontainers/runc
+WORKDIR /go/src/github.com/opencontainers/runc
+# Nov 18, 2020
+ARG RUNC_VERSION="689513cc0946d8e968abbe35544d5a7827d2bcfa"
```

Comment:
can we get this runc into containerd instead? we track the version that matches containerd currently and don't need to build it every time we build the base image.

Reply:
Yes, after the release of runc rc93. It will be very soon: https://github.com/opencontainers/runc/issues/2659

Reply:
[watching for this / subsrcribed, it doesn't seem _too_ far off and it would be preferable if possible]

Reply:
We have rc93 now